src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupTests.cs

```diff
@@ -62,15 +62,15 @@ public class BackupTests : BackupTestBase
             });
         }
 
-        [Fact(Skip="BRP not working.")]
+        [Fact]
         public void TestRestoreBackup()
         {
             RunTest((client) =>
             {
                 var backupLocation = "local";
                 var backup = client.BackupLocations.CreateBackup(ResourceGroupName, backupLocation);
                 Assert.NotNull(backup);
-                client.Backups.Restore(backupLocation, ResourceGroupName, backup.Name);
+                client.Backups.Restore(backupLocation, ResourceGroupName, backup.BackupId);
```

Comment:
You should use the name of the resource.  We should file a bug against the AutoRest team to handle nested resources correctly.

Reply:
Make sense. So you mean we should create an issue in Azure/autorest?

Reply:
Yeah I think so.  I will talk with Bala and the AutoRest team